lib/app/include/syscall.h

```diff
@@ -11,8 +11,11 @@
    defs */
 #define SYSCALL_OCALL 1001
 #define SYSCALL_SHAREDCOPY  1002
-#define SYSCALL_ATTEST_ENCLAVE  1003
-#define SYSCALL_EXIT  1101
+#define SYSCALL_ATTEST_ENCLAVE 1003
+#define RUNTIME_SYSCALL_SEND 1004
```

Comment:
For clarity and consistency, lets call these: SYSCALL_MAILBOX_SEND and SYSCALL_MAILBOX_RECV